tools/packaging/static-build/kernel/Dockerfile

```diff
@@ -2,9 +2,13 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+FROM mikefarah/yq:3.4.1 as build
```

Comment:
@Bevisy, I'd like to suggest a different approach here.

Instead of using a multi-stage build and end up depending on an image that may or may not be updated, or that may or may not be supported in different architectures, why not doing something like what's done on https://github.com/kata-containers/tests/blob/main/.ci/install_yq.sh?

Reply:
Reasonable. A simpler way to support multiple architectures makes more sense. The lack of multi-architecture image makes the use of multi-stage builds seem stupid, although I was planning to maintain a multi-architecture image of 3.4.1 yq.

Reply:
I actually like the a idea of using the multi-stage build but the lack of images for non-x86_64 architectures makes the solution complicated.

On https://github.com/kata-containers/kata-containers/issues/3086#issuecomment-975754430 I pasted a diff with changes that could be made to install yq with `install_yq.sh` script as suggested by @fidencio

Reply:
@wainersm @fidencio By looking at the normal github-action ci/kata-deploy, it is executing docker inside a docker environment with the yq tool prepared. So directly executing static-build will fail, but I don't feel the need to fix this failure because we are no longer using a context with the ci environment when the problem is found. In fact, what we did was to prepare yq in the image, so if it is your own local build, you can still consider using a multi-stage build, you just need to simply add two lines of dockerfile, but for ci execution, I think we can keep the status quo.